docs/nitclk/class.rst

```diff
@@ -183,7 +183,7 @@ finish_sync_pulse_sender_synchronize
             
 
 
-        :type sessions: list of (nimi-python Session class or nitclk.SessionReference)
+        :type sessions: (nimi-python Session class or nitclk.SessionReference)
```

Comment:
this is still wrong, should be "list of..."

Reply:
Missed in metadata. Fixed